openmetadata-ui/src/main/resources/ui/src/components/Glossary/GlossaryTerms/tabs/AssetsTabs.component.tsx

```diff
@@ -315,30 +314,6 @@ const AssetsTabs = forwardRef(
       setActiveEntity(data);
     }, [type, entityFqn]);
 
-    const tabs = useMemo(() => {
```

Comment:
Why we removed this?

Reply:
we were not using this in the file, and it was mentioned in the dependency only, that why it was not throwing the error.